src/renderers/native/__tests__/ReactNativeMount-test.js

```diff
@@ -58,4 +58,21 @@ describe('ReactNative', () => {
     expect(UIManager.updateView).toBeCalledWith(3, 'View', { foo: 'bar' });
   });
 
+  it('should be able to create and update a native component', () => {
+    var View = createReactNativeComponentClass({
+      validAttributes: { foo: true },
+      uiViewClassName: 'View',
+    });
+
+    var y;
+    var x = ReactNative.render(<View foo="foo" />, 11, function() {
+      y = this;
+    });
+
+    expect(x).toBe(y);
+    // We expose refs with a bunch of internal props. Yuck. But this should not
+    // be TopLevelWrapper.
+    expect(x._currentElement.props.foo).toBe('foo');
```

Comment:
It will fail regardless now but once we fix RN this could pass. Otherwise we later have to go through this test. 15 minutes saved later.